languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -15787,6 +15787,127 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rule id="MAKE_PRP_VBZ" name="makes me goes -> makes me go" default="temp_off">
+            <antipattern>
+                <token postag_regexp="yes" postag="SENT_START|PCT|CC|WP|PRP" />
+                <token min="0">as</token>
+                <token postag="PRP" />
+                <token min="0" max="2" postag_regexp="yes" postag="MD|RB" />
+                <token min="0" inflected="yes">do</token>
+                <token>see</token>
+                <token min="0" postag="PCT" />
+                <token>it</token><!-- subject or expletive -->
```

Comment:
In my experience, antipatterns don't handle multiple `min="0"` very well. They tend to only work when all optional tokens are present or absent. Maybe try `skip="1"` and an `exception scope="next" negate_pos="yes" postag="PRP"` for example.